pkg/lib/journal.css

```diff
@@ -101,7 +101,7 @@
 
 .empty-message {
   inline-size: 100%;
-  color: var(--pf-v5-global--Color--200);
+  color: var(--pf-v6-global--Color--200);
```

Comment:
This does not exists in PF6 and is replaced wrong everywhere

Reply:
It's only the three in this file. Most other places still even have `--pf-v5-global--Color..`